### DISCUSSION TITLE
Module not found: Error: Can't resolve './build/Release/obj.target/test

### DISCUSSION THREAD

**Discussion by xinsheng1964871232** [eclipse-theia/theia#8466] [General]
browser-app: Module not found: Error: Can't resolve './build/Release/obj.target/test 

I want to use C ,but have a question

**Comment by vince-fugnitto** [comment]
@xinsheng1964871232 can you please provide additional information so we can better help/understand?

> I want to use C ,but have a question

What is the question?

**Reply by xinsheng1964871232** [reply]
question： browser-app: Module not found: Error: Can't resolve './build/Release/obj.target/test

using bindings and node-gyp rebuild，I get a test.node. 
`//a.js
var tmp = require('bindings')('test');
console.log(tmp.test()) // result: 3

//in theia 
var tmp = require('bindings')('test');
//yarn prepare
//....
//browser-app: Module not found: Error: Can't resolve './build/Release/obj.target/test``

**Reply by vince-fugnitto** [reply]
I'm sorry but the question is still unclear.
If you are having issues building, please ensure that you have the proper [prerequisites](https://github.com/eclipse-theia/theia/blob/master/doc/Developing.md#prerequisites).
In addition, what are the steps to reproduce the problem?

**Reply by xinsheng1964871232** [reply]
Yeah， it is easy，
you can try。
// add.c
int add(){
        int a=1;
        int b=2;
        return a+b;
}
![捕获](https://user-images.githubusercontent.com/44309714/92325049-0ba10600-f07a-11ea-8c94-0eb59b347bfa.PNG)


https://github.com/xinsheng1964871232/11.git
